drivers/flash/spi_nor.c

```diff
@@ -101,6 +103,26 @@ struct spi_nor_config {
 	 * This information cannot be derived from SFDP.
 	 */
 	uint8_t has_lock;
+
+#if ANY_INST_HAS_T_ENTER_DPD || ANY_INST_HAS_DPD
+	uint16_t t_enter_dpd; /* in microseconds */
+	uint16_t t_dpdd_ms;   /* in microseconds */
+#endif
+
+#if ANY_INST_HAS_T_EXIT_DPD
+	uint16_t t_exit_dpd; /* in microseconds */
+#endif
```

Comment:
It does not make sense that a device would have enter DPD without exit DPD or vice versa, just make it one check and include both exit and enter parts

Reply:
Done